src/buildercore/trop.py

```diff
@@ -23,7 +25,8 @@
 osissuefn("embarassing code. some of these constants should be pulled form project config or given better names.")
 
 SECURITY_GROUP_TITLE = "StackSecurityGroup"
-EC2_TITLE = 'EC2Instance'
+EC2_TITLE = 'EC2Instance1'
```

Comment:
Reason: the code that creates the EC2 part of the template is parameterized with a for loop to create `EC2Instance%d`, so in the single instance case the DNS have to refer to `EC2Instance1`.

Then there is `ext`, the additional volume. That just needs to be extended to make multiple copies and be added to `EC2Instance%d`, but I didn't want to add even more changes so I started to support only multiple EC2 whose project does not have `ext`.